### PR TITLE
[DXEC-675] Redirect Social Connections to Marketplace

### PR DESCRIPTION
**Pull request by joshcanhelp**
- Redirects all active social connections to their Marketplace pages
- Redirects deprecated extensions to Marketplace listings
- Removes deprecated/removed connections from the list

Auto-formatter added a bunch of extra lines so turn off whitespace during the review with this link:
https://github.com/auth0/docs/pull/9765/files?diff=unified&w=1

All the redirects work as expected, not sure if we should also remove the old content so it's not confusing?

### CODE REVIEW

File: config/redirects.js [hunk 1]
```diff
@@ -1897,12 +2056,18 @@ module.exports = [
...
   },
   {
-    from: ['/extensions/gitlab-deploy','/extensions/gitlab-deployments'],
-    to: '/extensions'
+    from: [
+      '/extensions/gitlab-deploy',
+      '/extensions/gitlab-deployments'
+    ],
+    to: 'https://marketplace.auth0.com/integrations/gitlab-pipeline'
   },
```
Comment (joshcanhelp) [inline config/redirects.js:2114]: Everything below in this file is whitespace only

File: articles/connections/social/instagram.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ image: /media/connections/instagram.png
 seo_alias: instagram
 description: Learn how to add login functionality to your app with Instagram. You will need to generate keys, copy these into your Auth0 settings, and enable the connection.
 toc: true
-public: true
+public: false
```
Comment (joshcanhelp) [inline articles/connections/social/instagram.md:9]: Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.

File: articles/connections/social/pinterest.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ image: /media/connections/pinterest.png
 seo_alias: pinterest
 description: Learn how to add login functionality to your app with Pinterest. You will need to obtain a Client ID and Client Secret for Pinterest.
 toc: true
-public: true
+public: false
```
Comment (joshcanhelp) [inline articles/connections/social/pinterest.md:8]: Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.

File: articles/connections/social/soundcloud.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 title: Connect Apps to SoundCloud
 connection: SoundCloud
 toc: true
-public: true
+public: false
```
Comment (joshcanhelp) [inline articles/connections/social/soundcloud.md:5]: Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.

File: articles/connections/social/pinterest.md (hunk 3 above)
Comment (joshcanhelp) [reply articles/connections/social/pinterest.md:8]: The resulting page could use an update about the status of their API or just removed/404:

https://auth0.com/docs/connections/social/pinterest

**Review by rkhoriander — APPROVED** [review]

**Pull request merged**